capstone/src/main/webapp/youtubeScript.js

```diff
@@ -321,12 +331,12 @@ function perspectiveToxicityScale(attributeAverages) {
   const amountMoreThanKnoop = knoopScore - knoopLow;
   const mohsDecimal = amountMoreThanKnoop / knoopRange;
   const perspectiveToxicityScore = (mohsScore + mohsDecimal).toFixed(1);
-  document.getElementById('search-type').appendChild(document.createElement("br"));  
-  document.getElementById('search-type').append("Perspective Toxicity Score" + " : " + perspectiveToxicityScore);
+  document.getElementById('perspective-toxicity-score').innerHTML = ("Perspective Toxicity Score" + " : " + perspectiveToxicityScore);
```

Comment:
Is it possible to remove the parentheses?

Reply:
I've removed them in "address Daniel's and Chris's nits"